packages/devtools-components/src/tests/tree.js

```diff
@@ -446,14 +446,18 @@ function formatTree(wrapper) {
       let arrowStr = "  ";
       if (arrow.exists()) {
         arrowStr = arrow.hasClass("expanded") ? "▼ " : "▶︎ ";
-      } else {
-        arrowStr = "  ";
       }
-      return `${indentStr}${arrowStr}${node.text()}`;
+
+      return `${indentStr}${arrowStr}${getSanitizedNodeText(node)}`;
     })
     .join("\n");
 }
 
+function getSanitizedNodeText(node) {
+  // Stripping off the invisible space used in the indent.
+  return node.text().replace(/\u200B/g, "");
```

Comment:
`/^\u200B+/` would make it more intentional to only strip leading spaces.

Reply:
👍, DONE ✅